core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/loadbalancer/LoadBalancer.java

```diff
@@ -39,9 +41,16 @@
 
     private String endpoint;
 
+    /**
+     * @deprecated Use {@link #targetGroupSet} instead.
+     */
+    @Deprecated
     @OneToMany(mappedBy = "loadBalancer", cascade = CascadeType.REMOVE, orphanRemoval = true)
     private Set<TargetGroup> targetGroups = new HashSet<>();
```

Comment:
can't we just remove these fields from the entity and leave them in the database. As you wrote somewhere these are not really used in any environment yet. If we can't remove them please create a followup jira to remove them in the next release, if you haven't created one already. thanks

Reply:
because of continuous deployment, we need to keep backward and forward compatibility
so a follow-up jira is indeed needed to remove these.

Reply:
Right. Originally I had deleted them, but I had a talk with Arpad about the compatibility requirements and that led me to adding them back and marking them deprecated. I'll open another JIRA to clean these methods up once this PR is in.